react/features/base/dialog/components/native/InputDialog.js

```diff
@@ -116,7 +112,6 @@ class InputDialog<P: Props, S: State> extends AbstractDialog<P, S> {
                     <Dialog.Input
                         autoFocus = { true }
                         onChangeText = { this._onChangeText }
-                        underlineColorAndroid = { FIELD_UNDERLINE }
```

Comment:
Why did you remove this?

Reply:
We already have it set

 underlineColorAndroid={PlatformColor(
          `@android:color/${
            isDark ? "hint_foreground_dark" : "hint_foreground_light"
          }`
        )}